src/cli.py

```diff
@@ -49,21 +65,35 @@ def main():
         type=str,
         help='Key value pairs of parameters. E.g "Key1=Value1,Key2=Value2..."',
     )
+    parser.add_argument(
+        "-r",
+        "--root-path",
+        type=str,
+        help='The root path to append all file paths contained within the config file to. E.g "/foo/bar"',
+    )
     parser.add_argument(
         "--version",
         action="version",
         version="%(prog)s {version}".format(version=__version__),
     )
     args = parser.parse_args()
-    print(args)
+
+    # Determine root path
+    root_path = None
+    if args.root_path:
+        root_path = Path(args.root_path)
+    else:
+        root_path = get_root_path_from_env_var("DFM_ROOT_PATH")
 
     # Parse parameters
     if args.parameters:
         parameters = parse_parameter_string(args.parameters)
     else:
         parameters = None
     if args.action == "merge":
```

Comment:
could be a good situation for https://learnpython.com/blog/python-match-case-statement/

Reply:
Will go with your suggested one-liner for if else instead